libs/net/template/Models/Reports/TemplateModel.cs

```diff
@@ -13,7 +13,7 @@ public class TemplateModel : RazorEngineTemplateBase
     /// <summary>
     /// get/set - An array of content.
     /// </summary>
-    public IEnumerable<ContentModel> Content { get; set; }
+    public ContentModel[] Content { get; set; }
```

Comment:
This isn't technically the correct way to do this.  You are changing this because of how you must have implemented the templates.  An IEnumerable can be an array or it can be a collection.  If you want to treat it like an array, then use the `ToArray()` in your template.

Reply:
Updated, thanks